library/core/src/num/dec2flt/mod.rs

```diff
@@ -171,9 +171,26 @@ macro_rules! from_str_float_impl {
         }
     };
 }
+
+#[cfg(target_has_reliable_f16)]
+from_str_float_impl!(f16);
 from_str_float_impl!(f32);
 from_str_float_impl!(f64);
 
+// FIXME(f16_f128): A fallback is used when the backend+target does not support f16 well, in order
+// to avoid ICEs.
+
+// After the bootstrap bump this should be: `#[cfg(not(target_has_reliable_f16))`
```

Comment:
If true, please write this as a pair of cfg_attrs not a comment - we have some partial automation that can find and replace cfg attrs automatically but comments are near invisible to that.

Reply:
That is indeed not correct, the comment is from before the post-bump rebase. Removed.